code/modules/mob/living/simple_animal/friendly/robot_customer.dm

```diff
@@ -83,3 +85,7 @@
 		. += "<span class='notice'>Their order was: \"[attending_venue.order_food_line(ai_controller.blackboard[BB_CUSTOMER_CURRENT_ORDER])].\"</span>"
 
 
+
+/mob/living/simple_animal/robot_customer/Login()
+	. = ..()
+	status_flags &= ~GODMODE //owned admin
```

Comment:
if an admin wants to make a robot customer sapient, they probably want it to remain invincible anyway (and can remove the godmode flag if they don't)